api/src/main/resources/moodle.yaml

```diff
@@ -2,6 +2,7 @@
 # MOodle API information can be found at :
 # https://docs.moodle.org/dev/Web_service_API_functions
 
+#The host will get updated as per the environment it will be aoudev.nnlm.gov for dev and test
```

Comment:
I'd move this down to L13 where the host value is. And it may be worth referencing "ComplianceServiceImpl.java" as the place where this configuration happens.